### PR TITLE
graphql: add 4844 blob fields

### PR DESCRIPTION
**Pull request by jsvisa**

### CODE REVIEW

File: graphql/graphql.go [hunk 1]
```diff
@@ -468,6 +482,40 @@ func (t *Transaction) CumulativeGasUsed(ctx context.Context) (*hexutil.Uint64, e
...
+
+	receipt, err := t.getReceipt(ctx)
+	if err != nil || receipt == nil {
+		return nil, err
+	}
+	ret := hexutil.Uint64(receipt.BlobGasUsed)
+	return &ret, nil
+}
+
+func (t *Transaction) BlobGasPrice(ctx context.Context) (*hexutil.Big, error) {
```
Comment (s1na) [inline graphql/graphql.go:502]: This seems to be the same as maxFeePerBlobGas? at least here is being set this way: https://github.com/ethereum/go-ethereum/blob/5cf53f51ac556cfff2aee9d405efd336298a3aca/core/state_processor.go#L139

But not sure why the same field would be repeated in receipt and tx objects

File: graphql/graphql.go [hunk 2]
```diff
@@ -312,8 +310,6 @@ func (t *Transaction) MaxFeePerGas(ctx context.Context) *hexutil.Big {
 		return nil
 	}
 	switch tx.Type() {
-	case types.AccessListTxType:
```
Comment (s1na) [inline graphql/graphql.go:315]: I guess we need to add blob tx type here too?

File: graphql/graphql.go (hunk 1 above)
Comment (s1na) [reply graphql/graphql.go:502]: Ok no the receipt field is different. The line I shared might be a bug

File: graphql/graphql.go [hunk 3]
```diff
@@ -327,15 +323,33 @@ func (t *Transaction) MaxPriorityFeePerGas(ctx context.Context) *hexutil.Big {
...
```
Comment (s1na) [inline graphql/graphql.go:330]: Same here

**Review by s1na — CHANGES_REQUESTED** [review]
Please add `blobGasUsed` and `excessBlobGas` to the block.

File: graphql/graphql.go (hunk 1 above)
Comment (jsvisa) [reply graphql/graphql.go:502]: Haha, seems fixed in #28082

**Review by s1na — APPROVED** [review]
LGTM

**Review by holiman — APPROVED** [review]
LGTM

**Comment by holiman** [comment]
> Let's wait for the spec to be merged in first.

YOLO

**Pull request merged**